integrations/extensions/starter-kits/servicenow/readme.md

```diff
@@ -0,0 +1,61 @@
+# [ServiceNow Incident Report]
+
+## Background
+
+
+ServiceNow is a cloud-based software platform for IT Service Management (ITSM) which helps to automate IT Business Management.
+It has a variety of functionalities, however this extension will focus on Incident management, specifically creating an Incident.
+
+Our focus would be on [ServiceNow Table API](https://developer.servicenow.com/dev.do#!/reference/api/sandiego/rest/c_TableAPI) and specifically on using `POST /now/table/{tableName}` where `tableName` is `incident`.
+
+This Extension is also described in the following [blog](update link).
+
+## Pre-Requisite Steps
+
+ServiceNow offers a Personal Developer Instance. See details [here](https://developer.servicenow.com/dev.do#!/learn/learning-plans/rome/new_to_servicenow/app_store_learnv2_buildmyfirstapp_rome_personal_developer_instances).
```

Comment:
I think we should make it clear that this is required in order to do the next steps. Something like:
```suggestion
An instance is required to use ServiceNow APIs. Luckily, ServiceNow offers a Personal Developer Instance, well-suited for testing and demos. Follow the instructions [here](https://developer.servicenow.com/dev.do#!/learn/learning-plans/rome/new_to_servicenow/app_store_learnv2_buildmyfirstapp_rome_personal_developer_instances) to create an account and provision a Personal Developer Instance.
```